### PR TITLE
Add python package ptplot

### PR DESCRIPTION
**Pull request by AndrewRook**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AndrewRook** [comment]
Ok I think I sorted out the linux dependency conflicts and this should be ready. If you'd rather not have all of my troubleshooting commits merged let me know and I can start a new PR from a branch with just the latest commit.

File: recipes/ptplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+    - python >=3.7
+  run:
+    - bokeh >=2.3.3
+    - nodejs >=14.17.0
+    - numpy >=1.19.5
+    - pandas >=1.2.0
+    - patsy >=0.5.1
+    - python >=3.7
+    - svgpathtools >=1.4.1
```
Comment (saraedum) [inline recipes/ptplot/meta.yaml:29]:
```suggestion
    - bokeh >=2.3.3,<3
    - nodejs >=14.17.0,<15
    - numpy >=1.19.5,<2
    - pandas >=1.2.0,<2
    - patsy >=0.5.1,<0.6.0
    - python >=3.7
    - svgpathtools >=1.4.1,<2
```

I believe we usually add an upper bound to most dependencies, assuming that they use semantic versioning.

Comment (AndrewRook) [reply recipes/ptplot/meta.yaml]: Oh, ok sounds good. I committed the suggestion, thanks @saraedum!

File: recipes/ptplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - bokeh >=2.3.3,<3
+    - nodejs >=14.17.0,<15
```
Comment (ocefpaf) [inline recipes/ptplot/meta.yaml:24]: Is nodejs really needed at run time? I'm no expert, just curious.

Comment (AndrewRook) [reply recipes/ptplot/meta.yaml:24]: Hey, @ocefpaf, good question! I have a custom bokeh extension as part of the package, and [until I figure out how to prebuild it](https://github.com/bokeh/bokeh/discussions/11603) users will need to have node installed in order to have bokeh runtime-compile it. Most of my users won't have a working node install or the technical savvy to get one, so I figured the easiest way to deal with that was to make it a dependency.

**Comment by ocefpaf** [comment]
Nice package! I've been trying to get my young cousin, sports fan, to learn code and these pkgs are a great way to get his interest.

**Comment by AndrewRook** [comment]
@ocefpaf cool! There are more and more packages for this kind of thing. If you haven't seen it already, you might want to direct him to [pysport](https://pysport.org/), which has a collection of OSS libraries for sports analysis. 

Also key is finding good datasets, which can be nontrivial. I'm only really familiar with American football analytics, but if he likes the NFL then I'd recommend [NFLFastR's data](https://github.com/nflverse/nflfastR-data) for play-by-play data. The NFL also runs Kaggle contests with player tracking data, one of which [just kicked off](https://www.kaggle.com/c/nfl-big-data-bowl-2022). (The player tracking data is probably way harder for a beginner though, so that might not be a good starting point.)

**Pull request merged**